Packages/Netherlands3D/package.json

```diff
@@ -131,6 +131,15 @@
     "name": "3D Amsterdam Team",
     "url": "https://3d.amsterdam.nl"
   },
+  "scopedRegistries": [
```

Comment:
Weet je zeker dat dit kan in de package.json? Ik dacht dat dit een feature van de manifest.json was?

Reply:
ja je heb gelijk. gek genoeg staat daar ook al de eu.netherlands3d.web-cursors bij de dependencies..